docs/docs/learn/index.md

```diff
@@ -20,84 +20,88 @@ The learning path is structured as a progression through five key areas:
 
 **Module 1: Foundations** - Build your understanding of Aztec's core architecture, including its hybrid state model, privacy mechanisms using zero-knowledge proofs, and the unique execution environments (PXE and AVM).
 
-**Module 2: Smart Contract Development** - Master writing smart contracts in Noir, Aztec's domain-specific language. You'll learn to work with both private and public state, understand the execution flow between private and public contexts, and implement secure contract patterns.
+**Module 2: Smart Contract Development** - Learn how to write smart contracts in [Noir](https://noir-lang.org/), Aztec's domain-specific language. You'll learn to work with both private and public state, understand the execution flow between private and public contexts, and implement secure contract patterns.
 
-**Module 3: Security & Privacy** - Deep dive into privacy preservation techniques, understand common vulnerabilities in private smart contracts, and learn best practices for maintaining confidentiality while ensuring contract functionality.
+**Module 3: Security & Privacy** - Deep dive into writing secure Aztec smart contracts. Understand common vulnerabilities in private smart contracts, and learn best practices for maintaining privacy and security while ensuring contract functionality.
 
-**Module 4: Testing & Deployment** - Gain practical experience with the Aztec Sandbox, write comprehensive tests for private functions, deploy contracts to test networks, and debug privacy-preserving applications.
+**Module 4: Testing & Deployment** - Gain practical experience with the Aztec Sandbox, write comprehensive tests for private functions, deploy contracts to testnet, and debug privacy smart contracts.
 
-**Module 5: Advanced Patterns** - Explore sophisticated contract architectures, cross-contract private interactions, optimized proof generation, and integration with existing Ethereum infrastructure.
+- [TODO] add links for testnet and sandbox
+
+**Module 5: Advanced Patterns** - Explore advanced contract architectures, cross-contract private interactions, optimized proof generation, and integration with existing Ethereum infrastructure.
 
 ## The Aztec Stack
 
 Understanding Aztec requires familiarity with several interconnected components:
 
 ### Core Components
 
-**Private Execution Environment (PXE)** - Your client-side companion that executes private functions locally, manages keys and notes, and generates zero-knowledge proofs. The PXE ensures that sensitive computations never leave your device.
+**Private Execution Environment (PXE)** - Your client-side companion that executes private functions locally, manages keys and notes, and generates zero-knowledge proofs. The PXE ensures that sensitive computations never leave your device. It is a TypeScript library that can be run within Node.js, inside wallet software or a browser.
```

Comment:
I think we should update this section to mention that it is a core component of a wallet and is important to understand conceptually, but application developers should not be interfacing with the PXE directly